GMC_identification.ipynb

```diff
@@ -0,0 +1,1603 @@
+{
```

Comment:
<p>unclear which double hashes you want to include/not (can see this is a task in work as a temp fix used)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Good point. Will push a version without the commenting out</p>